lib/cirro_io_v2/resources/responses/gig_responce.rb

```diff
@@ -0,0 +1,15 @@
+module CirroIOV2
+  module Resources
+    module Responses
+      class GigResponce
+        GIG_PARAMS = %i[id object title description url start_at end_at total_seats invitation_mode
+                        filter_query tasks notification_payload epam_options].freeze
+
+        def self.create(body)
```

Comment:
can we maybe change `create` to `build` or `new`?
it is not creating anything it is really just a wrapper to the server response

Reply:
Then maybe wrap?

Reply:
Good point, I've renamed it to build